hbase-server/pom.xml

```diff
@@ -524,6 +524,44 @@
       <plugin>
         <artifactId>maven-antrun-plugin</artifactId>
         <executions>
+          <execution>
+            <!-- Copy static CSS and JS files for the UI. -->
+            <id>copy-css-js-to-ui</id>
+            <goals>
+              <goal>run</goal>
```

Comment:
Is it possible to remove this copy stage and instead place the resources as part of the unpack action below?

Reply:
Thanks, now I was able to do this finally. Only for this we have to use newer maven-dependency-plugin version.